config/initializers/locale.rb

```diff
@@ -0,0 +1,130 @@
+def locale_names
```

Comment:
This will create a method in global namespace and can create problems. create app/services/i18n_support.rb and add 
```ruby
class I18nSupport
  def self.locale_names
  ...
  end
end
```

Reply:
Also, no need to list all locales, just list supported locales, en and hi for now

Reply:
changed method to a class method!

limited number of locale to en and hi